with-sveltekit/src/routes/articles.json.js

```diff
@@ -0,0 +1,35 @@
+import 'dotenv/config';
+
+const { STEPZEN_ENDPOINT, STEPZEN_API_KEY } = process.env;
+
+export async function post() {
+  const response = await fetch(STEPZEN_ENDPOINT, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `apikey ${STEPZEN_API_KEY}`,
+    },
+    body: JSON.stringify({
+      query: `{
+        getArticles(username: "ajcwebdev", per_page: 100) {
```

Comment:
Can we make this a variable in the config file? We ask people for their Dev.to API key so it makes sense to also ask for their username. Fine to mention your username as a "default"